www/Media.js

```diff
@@ -35,16 +35,22 @@ var mediaObjects = {};
  * @param errorCallback         The callback to be called if there is an error.
  *                                  errorCallback(int errorCode) - OPTIONAL
  * @param statusCallback        The callback to be called when media status has changed.
- *                                  statusCallback(int statusCode) - OPTIONAL
```

Comment:
This line should probably not be moved down, right?

Reply:
ah that was a mistake!

Reply:
Fixed!